includes/wikia/tasks/Tasks/SiteWideMessagesTask.class.php

```diff
@@ -9,8 +9,14 @@
 
 
 use FluentSql\StaticSQL;
+use Wikia\DependencyInjection\Injector;
+use Wikia\UserGroups\UserGroupStorage;
 
 class SiteWideMessagesTask extends BaseTask {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "SiteWideMessagesTask" has 26 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)